composer.json

```diff
@@ -19,9 +19,9 @@
     "require-dev": {
         "aws/aws-sdk-php": "^2.4.9 || ^3.0",
         "doctrine/couchdb": "~1.0@dev",
-        "elasticsearch/elasticsearch": "^7",
```

Comment:
`ruflin/elastica` also has a dependency to `elasticsearch/elasticsearch`, and they don't have a 8.x release yet. How do you test the es8 branch in your pipeline?

Reply:
Yeah ruflin/elastica is blocking this from being testable, we'd need to remove it in the pipeline to be able to run the tests against elsaticsearch/elasticsearch v8